packages/account/build/webpack.config.js

```diff
@@ -53,7 +53,6 @@ module.exports = function (env) {
             'poi-poa-docs-submitted': 'Components/poi-poa-docs-submitted/poi-poa-docs-submitted',
             'reset-trading-password-modal': 'Components/reset-trading-password-modal',
             'risk-tolerance-warning-modal': 'Components/trading-assessment/risk-tolerance-warning-modal.jsx',
-            'self-exclusion': 'Components/self-exclusion',
```

Comment:
why is this removed?

Reply:
Removed this because the the component is not being exported anywhere outside of the package

Reply:
@sanjam-deriv does this affect the code splitting dynamics, can you check if the bundle will be created separately or not?

Reply:
Added the new route in the config